example/src/main/java/com/vimeo/android/networking/example/vimeonetworking/TestAccountStore.java

```diff
@@ -34,18 +35,20 @@ public TestAccountStore(Context context) {
 //        mAccountManager = AccountManager.get(context);
     }
 
+    @Nullable
     @Override
     public VimeoAccount loadAccount() {
         return AccountPreferenceManager.getClientAccount();
     }
 
     @Override
-    public void saveAccount(@NonNull VimeoAccount vimeoAccount) {
+    public void saveNonUserAccount(@NonNull VimeoAccount vimeoAccount) {
         AccountPreferenceManager.setClientAccount(vimeoAccount);
     }
 
     @Override
-    public void saveAccount(@NonNull VimeoAccount vimeoAccount, @NonNull String email) {
+    public void saveAuthenticatedUserAccount(@NonNull VimeoAccount vimeoAccount,
+                                             @NonNull String accountName) {
```

Comment:
It would help to just add a line comment to this method to indicate that in non-demo code, you'd probably want to store the account name as well.